observability/readme.md

```diff
@@ -27,6 +28,26 @@ Building observable systems enables development teams at CSE to measure how well
 
 Read on how Observability can be implemented in [Machine Learning](ml-observability.md) engagements effectively during Model tuning, experimentation and deployment.
 
+## Observability As Code
+
+As much as possible, configuration and management of observability assets such as cloud resource provisioning, monitoring alerts and dashboards must be managed as code. Infrastructure as Code is acheived using any one of Terraform / Ansible / ARM Templates
```

Comment:
Infrastructure as code or observability as code? IaC now comes a bit out of the blue for me.